OnlineDB/SiStripConfigDb/test/plugins/testSiStripConfigDb.cc

```diff
@@ -1,15 +1,47 @@
-
-#include "OnlineDB/SiStripConfigDb/test/plugins/testSiStripConfigDb.h"
-#include "FWCore/ServiceRegistry/interface/Service.h"
-#include "FWCore/MessageLogger/interface/MessageLogger.h"
-#include "DataFormats/SiStripCommon/interface/SiStripConstants.h"
-#include "DataFormats/SiStripCommon/interface/SiStripEnumsAndStrings.h"
-#include "OnlineDB/SiStripConfigDb/interface/SiStripConfigDb.h"
+// system includes
 #include <iostream>
 #include <iomanip>
 #include <sstream>
 #include <string>
 
+// user includes
+#include "DataFormats/SiStripCommon/interface/SiStripConstants.h"
+#include "DataFormats/SiStripCommon/interface/SiStripEnumsAndStrings.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "OnlineDB/SiStripConfigDb/interface/SiStripConfigDb.h"
+
+class SiStripConfigDb;
+
+/**
+   @class testSiStripConfigDb
+   @author R.Bainbridge
+   @brief Simple class that tests SiStripConfigDb service
+*/
+class testSiStripConfigDb : public edm::one::EDAnalyzer<> {
+public:
+  testSiStripConfigDb(const edm::ParameterSet&);
+  ~testSiStripConfigDb();
```

Comment:
should this be moved to the default destructor?

Reply:
no the destructor is not trivial:
https://github.com/cms-sw/cmssw/blob/9bd0dbd0474b5546afa1b8e4d3240989663d7135/OnlineDB/SiStripConfigDb/test/plugins/testSiStripConfigDb.cc#L42-L45

Reply:
ok thanks!